.idea/misc.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="EntryPointsManager">
+    <list size="4">
+      <item index="0" class="java.lang.String" itemvalue="org.terasology.entitySystem.event.ReceiveEvent" />
+      <item index="1" class="java.lang.String" itemvalue="org.terasology.entitySystem.systems.RegisterSystem" />
+      <item index="2" class="java.lang.String" itemvalue="org.terasology.logic.console.commandSystem.annotations.Command" />
+      <item index="3" class="java.lang.String" itemvalue="org.terasology.registry.In" />
+    </list>
+    <writeAnnotations>
+      <writeAnnotation name="org.terasology.registry.In" />
+    </writeAnnotations>
+  </component>
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="liberica-11" project-jdk-type="JavaSDK" />
```

Comment:
project-jdk-name - machine specified parameter

Reply:
I think this is okay too. It specifies which JDK to use for the project, but it doesn't encode any machine-specific path to exactly where "liberica-11" is installed for you, so it can be whatever JDK you have with that name.

You can download and install the liberica distributions directly from the add JDK menu. Hopefully I left it with the default name it gives when you do that?

but this is definitely a good thing to get feedback on, as it really will be a mess if it doesn't work the way I think it does